cli/src/error.rs

```diff
@@ -1,7 +1,9 @@
 use thiserror::Error;
+ 
+pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
I don't think result alias shouldn't be defined inside of the error module. If I remember correctly that guy in the video had a "main" module? Maybe I'm misremembering.

Reply:
nvm